lib/internal/Magento/Framework/Setup/LoggerInterface.php

```diff
@@ -52,4 +52,12 @@ public function logInline($message);
      * @return void
      */
     public function logMeta($message);
+
+    /**
+     * Logs meta information in the current line
+     *
+     * @param string $message
+     * @return void
+     */
+    public function logMetaInline($message);
```

Comment:
I prefer to introduce such changes in backward compatible way - introduce the new interface and implementation for that.

Reply:
Sure, my biggest problem with introducing new interfaces is: how do you name them? @sidolov: do you have a suggestion?

Reply:
`LogMetafataInlineInterface` with method `execute`?

Reply:
I got your point but still, we will have a small percent of developers who implemented the interface and their implementation will be broken after such changes. That's the reason to have strict rules - to support 3d party code and make it reliable when you are using the core code marked as @api.

Reply:
Hi @sidolov 

I know the reasoning, but I very often see exceptions being allowed where no extra interfaces are required.
We already discussed this before in https://github.com/magento/magento2/pull/26748#issuecomment-593558031

Thanks